managed-databases/mongodb/how-to/delete-a-database-instance.mdx

```diff
@@ -0,0 +1,37 @@
+---
+meta:
+  title: How to delete a MongoDB™ Database Instance
+  description: This page explains how to delete a Database
+content:
+  h1: How to delete a MongoDB™ Database Instance
+  paragraph: This page explains how to delete a Database
+tags: managed-database postgresql ferretdb mongodb document database-instance
+dates:
+  validation: 2024-02-08
+  posted: 2023-08-01
+categories:
+  - managed-databases
+---
+
+<Macro id="requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [MongoDB™ Database Instance](/managed-databases/document-database/quickstart)
+</Message>
+
+1. Click **MongoDB™** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll down the page to the **Delete Instance** section and click **Delete Instance**. A pop-up appears to inform you that:
+    <Message type="important">
+      This will permanently destroy your Instance, and all your data will be lost. This action is irreversible.
+    </Message>
+4. Type **DELETE** to confirm and click **Delete this Instance**.
+    <Message type="tip">
+      Alternatively, you can delete your Instance from the Database Instances list by clicking <Icon name="more" /> next to the Instance name and then **Delete**.
+    </Message>
+
+
+<Navigation title="See also">
+  <PreviousButton to="/managed-databases/document-database/how-to/delete-database">How to delete a database</PreviousButton>
+</Navigation>
```

Comment:
```suggestion
```

Reply:
Previous and next pages are now automatically handled (defined by the order in the nav menu)